src/Stack/Build/ConstructPlan.hs

```diff
@@ -52,6 +52,7 @@ import           System.Environment (lookupEnv)
 import           System.IO (putStrLn)
 import           RIO.PrettyPrint
 import           RIO.Process (findExecutable, HasProcessContext (..))
+import Stack.Types.Dependency (DepValue(DepValue), DepType (AsLibrary))
```

Comment:
Please could this `import` line adopt the same format as used elsewhere in the module.